shared/src/libs/enums/exception-rule.enum.ts

```diff
@@ -0,0 +1,5 @@
+const ExceptionRule = {
+  ALLOWED_FILE_EXTENSIONS: 'PNG, JPEG or MPEG',
```

Comment:
We need to have only one place with storing config for allowed file extensions
server-application file has extensions: [ContentType.JPEG, ContentType.PNG, ContentType.MPEG]. It would be helpful on change that extensions

Reply:
Done